resolver/src/main/java/io/netty/resolver/DefaultHostsFileEntriesResolver.java

```diff
@@ -67,13 +87,26 @@ String normalize(String inetHost) {
         return inetHost.toLowerCase(Locale.ENGLISH);
     }
 
-    private static HostsFileEntries parseEntries() {
+    private static List<InetAddress> allAddresses(List<InetAddress> a, List<InetAddress> b) {
+        List<InetAddress> result = new ArrayList<InetAddress>(a);
```

Comment:
imho it would be better to use `new ArrayList<InetAddress>(a.size() + (b == null ? 0 : b.size()));`. Otherwise you may need to expand the array after creation

Reply:
fixed